client-common/src/main/java/org/apache/cassandra/sidecar/common/response/OperationsJobsResponse.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response;
+
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.cassandra.sidecar.common.utils.OperationsJobResult;
+
+/**
+ * Response structure of the operations jobs API
+ */
+public class OperationsJobsResponse
+{
+    private final UUID jobId;
+    private final OperationsJobResult.OperationsJobStatus status;
+    private final String operation;
+    private final String reason;
```

Comment:
Not sure I understand. Are you asking why we're not using `OperationsJobResult` as a member here?  

If so, I did not see a reason to have the hierarchy since all these are attributes of the jobs API response.

Reply:
`OperationsJobsResponse.status.operation == OperationsJobsResponse.operation` and `OperationsJobsResponse.status.reason == OperationsJobsResponse.reason`. 

They are duplicated.

Reply:
Latest revision removes the need for status and reason as they are inferred from the Future state which is cached.